parity-util-mem/src/allocators.rs

```diff
@@ -95,6 +99,13 @@ cfg_if! {
 			jemallocator::usable_size(ptr)
 		}
 
+	} else if #[cfg(feature = "mimalloc-global")] {
+
+		/// Use of mimalloc usable size C function through mimallocator crate call.
+		pub unsafe extern "C" fn malloc_usable_size(ptr: *const c_void) -> usize {
+			mimalloc_sys::mi_usable_size(ptr as *mut _)
```

Comment:
mimalloc doesn't actually mutate the value `ptr` points to, but requires a mut pointer in the API

Reply:
That sounds like something we should put in a comment in the code.